clippy_lints/src/utils/conf.rs

```diff
@@ -152,6 +152,8 @@ define_Conf! {
     (too_many_lines_threshold, "too_many_lines_threshold", 100 => u64),
     /// Lint: LARGE_STACK_ARRAYS. The maximum allowed size for arrays on the stack
     (array_size_threshold, "array_size_threshold", 512_000 => u64),
+    /// Lint: VEC_BOX. The minimum required size for boxed type in Vec
```

Comment:
```suggestion
    /// Lint: VEC_BOX. The size of the boxed type in bytes, where boxing in a `Vec` is allowed
```

Is 4kB based on something?

Reply:
Page size; but as I said in the pr, you are welcome to put any number you like

Reply:
Page size sounds good.